components/editor/ftd/ftd.grammar

```diff
@@ -0,0 +1,12 @@
+@top ImportCommand { "--" "import" ":" Name }
```

Comment:
You can create p1 level grammar only, try to port our old grammar, instead of creating full parser.

Reply:
I would love full parser, but it would be more work. Also the language support for fastn should be in fastn-stack org, so others can also use it.